db/seeds.rb

```diff
@@ -5,3 +5,23 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'nokogiri'
+require 'open-uri'
+require 'date'
+data = "http://lingvotutor.ru/230-samyx-populyarnyx-anglijskix-fraz-en-ru"
+doc = Nokogiri::HTML(open(data))
+original = []
+translated = []
+doc.css('#tablepress-608').map do |column|
+  column.css('.column-1').each do |i|
+    original << i.text.strip
+  end
+  column.css('.column-2').each do |j|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.